client/my-sites/plan-features/header.jsx

```diff
@@ -366,7 +405,7 @@ PlanFeaturesHeader.propTypes = {
 };
 
 PlanFeaturesHeader.defaultProps = {
-	currencyCode: 'USD',
+	currencyCode: null,
```

Comment:
Do you want the default to be explicitly `null`, or could we just omit it from the set of `defaultProps` ?